docs/charter.md

```diff
@@ -62,6 +64,11 @@ The _CPANSec_ intends to be a forum for coordinating and assisting in resolving
 The _CPANSec_ concerns itself with a limited domain:
 
 - Distributions published on CPAN, not including dual-life core modules;
+- CPAN distributions that function as shims (wrappers) for non-CPAN projects or resources. Examples:
+  - CPAN Distributions that are pure wrappers around compiled libraries (e.g. XS modules wrapping `libxml2` or `sqlite3`)
+  - CPAN Distributions (outside the `Alien::` namespace) that embed or use third-party (non-CPAN) projects or APIs directly.
+  - The `Alien::` distribution namespace
+  - Mozilla::CA
```

Comment:
```suggestion
```

Mozilla::CA is now handled by the [libwww-perl](https://github.com/libwww-perl) team, and semi-automatically updated. We probably don't need to specifically list it as a concern.

Reply:
Ok! I'll push an update.